### ISSUE TITLE
Move table based configuration to web.xml param

### ISSUE THREAD

**Issue by d3ep4k** [metamug/mason#48]
Initialize 

```xml
<servlet>
     <servlet-name>Router</servlet-name>
     <init-param>
         <param-name>MTG_AUTH_BASIC</param-name>
         <param-value>
         <![CDATA[
             select r.user_id,r.role_name 
             from usr_role r inner join usr u on r.user_id=u.user_id 
             WHERE u.user_name=$user AND u.pass_word=$pass
         ]]>
         </param-value>
     </init-param>
     <init-param>
         <param-name>MTG_AUTH_BEARER</param-name>
         <param-value>
         <![CDATA[
             SELECT r.user_id as sub,r.role_name as aud 
             FROM usr_role r inner join usr u on  r.user_id=u.user_id 
             WHERE u.user_name=$user AND u.pass_word=$pass
         ]]>
         </param-value>
     </init-param>
</servlet>
```

https://stackoverflow.com/a/28393315/1097600

**Comment by d3ep4k** [comment]
Since everything is routed through the router. It makes sense for the router to hold the init params instead of using context-params.

**Comment by d3ep4k** [comment]
Wondering are there any table dependencies. There should be none. There is a single DAO file. The config needs to change.

https://github.com/metamug/mason/blob/develop/src/main/java/com/metamug/mason/dao/AuthDAO.java

**Comment by kaisteel** [comment]
With web.xml based configuration it has its own pros and cons
Pros:
We can change auth scheme query even after the webapp is exported out of console.
Cons:
It kills the purpose of having a console for managing the backend.
Either way it won't need server restart the webapp will be redeploy automatically.
Also not to nitpick, but our router is a filter not a servlet

**Comment by ancode4** [comment]
The only table dependency is mtg_config. The usr and usr_role tables can be replaced by user tables existing in the connected DB.

Also, our goal is to keep Mason independent of the console, so let's avoid bringing the console in these discussions.

**Comment by ancode4** [comment]
Closed in dd793ba
